src/clj/rems/db/entitlements.clj

```diff
@@ -101,11 +116,13 @@
   :start (scheduler/start! process-outbox! (.toStandardDuration (time/seconds 10)))
   :stop (scheduler/stop! entitlement-poller))
 
-(defn- add-to-outbox! [action entitlements]
+(defn- add-to-outbox! [action type entitlements config]
```

Comment:
Could look up the type from the config here, right? (and default to `:basic` for no config). Just an idea.

Reply:
I created this further issue because there's other options to do too (e.g. not use the type in the values but in the outbox).

https://github.com/CSCfi/rems/issues/2684